test/e2e/storage/volume_metrics.go

```diff
@@ -170,13 +178,18 @@ var _ = utils.SIGDescribe("[Serial] Volume metrics", func() {
 	// Test for pv controller metrics, concretely: bound/unbound pv/pvc count.
 	Describe("PVController", func() {
 		const (
-			classKey     = "storage_class"
-			namespaceKey = "namespace"
-
-			boundPVKey    = "pv_collector_bound_pv_count"
-			unboundPVKey  = "pv_collector_unbound_pv_count"
-			boundPVCKey   = "pv_collector_bound_pvc_count"
-			unboundPVCKey = "pv_collector_unbound_pvc_count"
+			classKey         = "storage_class"
+			namespaceKey     = "namespace"
+			operationNameKey = "operation_name"
+			pluginNameKey    = "plugin_name"
+
+			boundPVKey              = "pv_collector_bound_pv_count"
+			unboundPVKey            = "pv_collector_unbound_pv_count"
+			failedPVKey             = "pv_controller_failed_pv_count"
+			boundPVCKey             = "pv_collector_bound_pvc_count"
+			unboundPVCKey           = "pv_collector_unbound_pvc_count"
+			volumeOperationKey      = "volume_operation_total_seconds"
+			volumeOperationErrosKey = "volume_operation_total_errors"
 		)
```

Comment:
Does the test scenario actually has errors for these metrics to be recorded? We should run these tests in a loop for 20-30mins before we merge, they are known to flake unless carefully verified.

Reply:
@gnufied, how to you run these tests?

I just noticed that they are skipped [0]. Also, I haven't had success running them locally; one of them always fail, and the other always skip.

[0] https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/63601/pull-kubernetes-e2e-gce/40853/